browser/BUILD.gn

```diff
@@ -191,6 +190,7 @@ source_set("browser_process") {
       "importer",
       "infobars",
       "//brave/app:brave_generated_resources_grit",
+      "//brave/browser/themes",
```

Comment:
why this change? Usually relative paths are used for subdirs

Reply:
Fixed to `themes`.